dask/dataframe/reshape.py

```diff
@@ -268,12 +276,36 @@ def pivot_table(df, index=None, columns=None, values=None, aggfunc="mean"):
             chunk_kwargs=kwargs,
         )
 
+    if aggfunc == "first":
+        pv_first = apply_concat_apply(
+            [df],
+            chunk=methods.pivot_first,
+            aggregate=methods.pivot_agg_first,
+            meta=meta,
+            token="pivot_table_first",
+            chunk_kwargs=kwargs,
+        )
+
+    if aggfunc == "last":
+        pv_last = apply_concat_apply(
+            [df],
+            chunk=methods.pivot_last,
+            aggregate=methods.pivot_agg_last,
+            meta=meta,
+            token="pivot_table_last",
+            chunk_kwargs=kwargs,
+        )
+
     if aggfunc == "sum":
         return pv_sum
     elif aggfunc == "count":
         return pv_count
     elif aggfunc == "mean":
         return pv_sum / pv_count
+    elif aggfunc == "first":
+        return pv_first
+    elif aggfunc == "last":
+        return pv_last
```

Comment:
I think the chain of if statements above is just to avoid duplicating the creation logic for `sum`/`count` in `mean` as well (although that could have been done a bit clearer). Since `pv_first`/pv_last` aren't ever re-used, I'd find it clearer to just move the `apply_concat_apply` calls for those down here and return them directly.

Reply:
Fully agree, thanks. Changed.